src/test/scala/io/confluent/examples/streams/GenericAvroScalaIntegrationTest.scala

```diff
@@ -101,7 +101,7 @@ class GenericAvroScalaIntegrationTest extends AssertionsForJUnit {
     }
 
     val stream: KStream[String, GenericRecord] = builder.stream(inputTopic)
-    stream.to(stringSerde, genericAvroSerde, outputTopic)
+    stream.to(outputTopic, Produced.`with`(stringSerde, genericAvroSerde))
```

Comment:
Question: Why do we need the backticks around `with`? (There are several others places in this PR where use these backticks, too.)

Reply:
it doesn't compile without it. I guess `with` is a keyword in scala

Reply:
Yeah, you are right -- `with` is a keyword in Scala (I didn't realize that the only instances in the PR with backticks was for Scala files).  So that's yet another downside of the current Scala interoperability for Kafka Streams, doh. :-(